content/using/install.md

```diff
@@ -321,7 +321,7 @@ Setting up and running a star or galaxy is a little bit different from setting u
 Stars are valuable cryptographic assets. They can be bought in one of two ways.
 
 - Transacting privately with a star owner.
-- Buying a star on a marketplace like [OpenSea](https://opensea.io/collection/urbit-id)
+- Buying a star on a marketplace like [OpenSea](https://opensea.io/collection/urbit-id) or [Planet Market](https://planet.market)
```

Comment:
I don't see where you can buy a star on this page?

Reply:
Planet Market sells stars via their OTC desk.  This is typically advertised on the @planet_market Twitter account.

Reply:
This isn't even [in the FAQ](https://planet.market/faq) though. It should perhaps be on the site somewhere?

Reply:
Good point.